karma.conf.js

```diff
@@ -65,10 +65,33 @@ module.exports = function(config) {
 		// if true, Karma captures browsers, runs the tests and exits
 		singleRun: true,
 
-		browserify: {
-			debug: true,
-			transform: [ 'debowerify' ]
-		}
-
-	});
+        coverageReporter: {
+            dir : 'test/coverage/',
+            reporters: [
+                { type: 'html',
+                    subdir: function(browser) {
+                        return browser.toLowerCase().split(/[ /-]/)[0];
+                    },
+                    watermarks: {
+                        statements: [75, 85],
+                        lines: [75, 85],
+                        functions: [75, 85],
+                        branches:[75, 85]
+                    }},
+                { type: 'json-summary', subdir: '.', file: 'summary.json' },
+            ]
+        }
+	};
+
+    //capture browserify config set in package.json to help prevent code duplication
+    var pkg = require('./package.json');
+    karmaConfig.browser = pkg.browser || {};
+    karmaConfig["browserify-shim"] = pkg["browserify-shim"] || {};
+    karmaConfig.browserify = pkg.browserify || {};
+    if (karmaConfig.browserify.transform) {
```

Comment:
the origami build always uses debowerify so you can safely assume browserify.transform exists